README.md

```diff
@@ -25,6 +25,22 @@ def deps do
 end
 ```
 
+## License
+
+Copyright 2022 JOSHMARTIN GmbH
```

Comment:
@maennchen should we update the copyright here since a significant portion of this code came from Gettext?

Reply:
@whatyouhide True. I didn‘t think about that because the initial implementation did not build on the yacc parser.

Something like

> Copyright 2015 Plataformatec Copyright 2020 Dashbit 2022 JOSHMARTIN GmbH